### ISSUE TITLE
Failed to start API Mock endpoint InternalFailure: The request processing has failed because of an unknown error, exception or failure.

### ISSUE THREAD

**Issue by hutber** [aws-amplify/amplify-cli#5641]
When I try to start my local mock service I will recieve the follow message:

```sh
GraphQL schema compiled successfully.

Edit your schema at /var/www/unsal.co.uk/amplify/backend/api/unsalcouk/schema.graphql or place .graphql files in a directory at /var/www/unsal.co.uk/amplify/backend/api/unsalcouk/schema
Failed to start API Mock endpoint InternalFailure: The request processing has failed because of an unknown error, exception or failure.
```
This has only been a problem in the last few days. Removing amplify completely and doing a `amplify init` will temporarily fix the issue. I have not been able to work out why/how this error occurs or the steps to make it happen.

4.29.7

**To Reproduce**
It is not clear to me on how to reproduce this currently.

I would expect to be able to start the mock server or display an error that is actionalible. 

Linux Mint 19.2
Node: v13.8.0

**Comment by akshbhu** [comment]
Hi @hutber 

Can you try with latest version to see if this issue persists? From the error, it looks like issue with DynamoDb Local.
Also can you tell which java version you are using and what categories you added so that I can also reproduce this on my end

**Comment by hutber** [comment]
Thanks @akshbhu 

To fix this, currently I just ended up removing the amplify folder completely and pulled down from AWS. This did fix the issue, I've had it come back a few times, so when that happens again. It may well be that I don't have the error again since moving to the newer version :)

I renamed the folders, so I still have those saved locally. I can upload them to a drive if you'd access to them?

```
openjdk version "11.0.8" 2020-07-14
OpenJDK Runtime Environment (build 11.0.8+10-post-Ubuntu-0ubuntu118.04.1)
OpenJDK 64-Bit Server VM (build 11.0.8+10-post-Ubuntu-0ubuntu118.04.1, mixed mode, sharing)
```
